### ISSUE TITLE
tf.keras api for autoencoders

### ISSUE THREAD

**Issue by ed-ortizm** [ed-ortizm/autoencoders#1]
Inherit from keras.models and keras.losess to create custom loss and custom layer
